ci/apps/java/src/main/resources/secrets.yml

```diff
@@ -0,0 +1,2 @@
+SPACE_USERNAME: space_username #!var secrets/username
```

Comment:
Do these need to be un-hardcoded? Same for the other apps.

Reply:
No, the fixed value is desired here. I left the comment in just as a way of documenting what would normally be here.

I'm not trying to test the buildpack secrets retrieval in these integration tests, just that the `conjur-env` binary gets sourced when the conjur buildpack is used with the different language buildpacks. 

So if these summon values show up in the final app, we know the conjur buildpack interacts correctly with the final buildpack.